scripts/core/editor3/reducers/editor3.jsx

```diff
@@ -54,6 +54,26 @@ export const forceUpdate = (state) => {
     };
 };
 
+
+function noCharactersAreSelectedAndCursorIsInsideCommentOrAnnotation(editorState) {
+    var selection = editorState.getSelection();
+    var character = editorState
+        .getCurrentContent()
+        .getBlockForKey(selection.getAnchorKey())
+        .getCharacterList()
+        .get(selection.getAnchorOffset());
+
+    if (character === undefined) {
+        return false;
+    }
+
+    var characterStyle = character.getStyle();
+
+    return selection.getAnchorKey() === selection.getFocusKey()
+        && selection.getAnchorOffset() === selection.getFocusOffset()
```

Comment:
this seems like [selection.isCollapsed()](https://draftjs.org/docs/api-reference-selection-state.html#iscollapsed)

Reply:
Yeah, it seems so. Anyway, in the meantime, I've found a better solution, so I'll close this one.